torch/functional.py

```diff
@@ -821,6 +822,22 @@ def btriunpack(LU_data, LU_pivots, unpack_data=True, unpack_pivots=True):
                      unpack_data=unpack_data, unpack_pivots=unpack_pivots)
 
 
+def btrisolve(b, LU_data, LU_pivots, out=None):
+    r"""Solves the system of equations :math:`Ax = b` using the partially pivoted LU
+    factorization of :math:`A` given by :attr:`LU_data` and :attr:`LU_pivots`.
+
+    For more information regarding :func:`torch.btrisolve`, please check
+    :func:`torch.lu_solve`.
+
+    .. warning::
+        :func:`torch.btrisolve` is deprecated in favour of :func:`torch.lu_solve` and will be
+        removed in the next release. Please use :func:`torch.lu_solve` instead.
+    """
+    warnings.warn("torch.btrisolve is deprecated in favour of torch.lu_solve and will be "
+                  "removed in the next release. Please use torch.lu_solve instead.", stackleve=2)
```

Comment:
this needs to be changed to `stacklevel`

Reply:
(I made the change in-place)

Reply:
Oh, I'm sorry about that. Thank you for fixing it.